static/js/pages/PodcastFrontpage_test.js

```diff
@@ -48,4 +53,47 @@ describe("PodcastFrontpage tests", () => {
       wrapper.find("PodcastCard").map(card => card.prop("podcast"))
     ).map(podcasts => assert.deepEqual(...podcasts))
   })
+
+  describe("loader", () => {
+    [
+      [true, true, true, true],
+      [true, false, true, true],
```

Comment:
I think this line should be `true, false, true, false`, but when I change it to that locally the test stops passing

Reply:
The episode rendering looks up information in both the episode and podcasts API data, so if only the podcast data is loading, the episode list still needs to show a spinner

Reply:
ahhh that makes sense